Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java

```diff
@@ -437,4 +461,22 @@ public void onUpdate(String entityId, Calendar lastSyncTime, boolean isSynced) {
             mPreferences.edit().putLong(entityId, lastSyncTime.getTimeInMillis()).apply();
         }
     }
+
+    private class VerificationListener implements AccountVerificationWatcher.VerificationStateListener {
+        @Override
+        public void onUpdate(AccountVerificationWatcher.Status status) {
+            switch (status) {
+                case VERIFIED:
+                    dismissReviewAccountDialog();
+                    return;
+
+                case SENT_EMAIL:
+                    showWaitingOnEmailConfirmation();
+                    return;
+
+                case UNVERIFIED:
+                    showUnverifiedAccount();
```

Comment:
are these wrapper functions necessary or could we inline their calls, given that this all happens from inside `Simplenote` now?

Reply:
I would leave them since they make the code clearer, but I see your point since they are just private methods and not part of the public interface.

Reply:
> since they make the code clearer

:trollface: well, they make me jump around the file to see what they do 🤷‍♂️

Reply:
thumbs down?